com.opera.Opera.yaml

```diff
@@ -90,6 +84,8 @@ modules:
       - bsdtar -xf /app/tmp-install/data.tar.xz -C /app/tmp-install/data
       - mv /app/tmp-install/data/usr/lib/x86_64-linux-gnu/opera /app
       - rm -rf /app/tmp-install opera.deb
+      - mv /app/opera/libffmpeg.so /app/opera/libffmpeg.so.opera
```

Comment:
Will Opera actually use the libffmpeg.so.opera file? If not, might as well `rm` it.

Reply:
Alright, that is easy